mock-plugin/src/test/java/io/mosip/esignet/plugin/mock/service/MockAuthenticationServiceTest.java

```diff
@@ -44,6 +47,9 @@ public class MockAuthenticationServiceTest {
     @Mock
     KeymanagerService keymanagerService;
 
+    @Mock
+    ObjectMapper objectMapper;
```

Comment:
@pvsaidurga Lets not mock objectMapper in unit test cases

Reply:
Okay @ase-101